src/Controls/StateControl.js

```diff
@@ -324,24 +324,33 @@ class StateControl extends THREE.EventDispatcher {
         viewCoords.copy(this._view.eventToViewCoords(event));
 
         switch (event.pointerType) {
-            case 'mouse':
+            case 'mouse': {
                 this._currentMousePressed = event.button;
-
+                let clickKeyPressed = this._currentKeyPressed;
+
+                if (clickKeyPressed === undefined) {
+                    if (event.ctrlKey) {
+                        clickKeyPressed = CONTROL_KEYS.CTRL;
+                    } else if (event.shiftKey) {
```

Comment:
Could you please add support to the third mouse modifier, i.e. the `meta` key (and add a `META` property in `CONTROL_KEYS` mapping to the code of one of the meta key ?

```suggestion
                        clickKeyPressed = CONTROL_KEYS.CTRL;
                    } else if (event.metaKey) {
                        clickKeyPressed = CONTROL_KEYS.META;
                    } else if (event.shiftKey) {
```

Reply:
done